test-assets/my-university/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Ideally these file names are prefixed with the relevant SC, and not with in nested folders. 

In this case, it could be

`test-assets/SC2-4-9-rule-name-my-test-asset-file.html`

Reply:
@jkodu, can we create some guidance on this? And maybe a clean-up... I just had a look in the folder and tried to copy what I saw.

Reply:
Thinking further about it, wouldn't it make sense that each rule had it's own folder for test assets, where the folder name is the same as the rule name?

Reply:
We reached a consensus on this in a previous meeting.